devices/samsung-dreamlte/kernel/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  mobile-nixos
+, fetchFromGitHub
+, fetchpatch
+, ...
+}:
+
+mobile-nixos.kernel-builder-gcc49 {
+  # From PostmarketOS, but with:
+  # CMDLINE_DROP_DANGEROUS_ANDROID_OPTIONS=y
```

Comment:
We don't really need to state such things. And anyway the more validation we'll be doing with our fresh tooling, the less true it will be.

Furthermore, eventually I want to move to `defconfig` snippets with only the minimum required options being defined, such that we don't need a fully-realised config file sitting in the repo.